operacje_na_listach/Function_for_exponentiation.py

```diff
@@ -0,0 +1,13 @@
+def power(list):
+    square=[]
+    for i in list:
+        square.append(i**2)#potęgowanie liczb
+    return square
+print(power([2, 4, 6, 8]))
+
+def power2(list, exponent):
+    any=[]
```

Comment:
`any` is also a built-in python function and you shouldn't name your variables like that